apps/test/unit/templates/CertificateTest.js

```diff
@@ -7,30 +7,35 @@ import responsive from '@cdo/apps/code-studio/responsiveRedux';
 
 describe('Certificate', () => {
   const store = createStore(combineReducers({responsive}));
+  let storedWindowDashboard;
+
+  beforeEach(() => {
+    storedWindowDashboard = window.dashboard;
+    window.dashboard = {
+      CODE_ORG_URL: 'https://code.org'
+    };
+  });
+
+  afterEach(() => {
+    window.dashboard = storedWindowDashboard;
+  });
 
   it('renders a Minecraft certificate for new Minecraft tutorials', () => {
     const wrapper = shallow(
       <Certificate tutorial="minecraft" isMinecraft={true} />,
       {context: {store}}
     ).dive();
-    expect(
-      wrapper
-        .find('img')
-        .html()
-        .includes('MC_Hour_Of_Code_Certificate')
+    expect(wrapper.find('img').html()).to.include(
+      'MC_Hour_Of_Code_Certificate'
     );
   });
 
   it('renders a Minecraft certificate for older Minecraft tutorials', () => {
-    const wrapper = shallow(
-      <Certificate type="minecraft" isMinecraft={true} />,
-      {context: {store}}
-    ).dive();
-    expect(
-      wrapper
-        .find('img')
-        .html()
-        .includes('MC_Hour_Of_Code_Certificate')
+    const wrapper = shallow(<Certificate isMinecraft={true} />, {
+      context: {store}
+    }).dive();
+    expect(wrapper.find('img').html()).to.include(
+      'MC_Hour_Of_Code_Certificate'
```

Comment:
"new" and "older" in these tests are unclear to me at this point because "new" relative to when? when the test were written?  If I recall correctly, the distinction for certificates is just that the Aquatic certificate is blue but all the other Minecraft versions are the same. I think the tests we want are : 
1.) renders a generic Minecraft certificate regardless of tutorial (no tutorial, minecraft any version except aquatic, or any other tutorial) if `isMinecraft={true}`
 2.) renders an aquatic specific tutorial if `isMinecraft={true}` && `tutorial='aquatic` =>  'MC_Hour_Of_Code_Certificate_Aquatic'

Reply:
An excellent point!  Since I included a behavior change here, I'll update with the suggested tests before merge.  Thanks for catching this!